### DISCUSSION TITLE
How to deal when vacuum goes unavailable?

### DISCUSSION THREAD

**Discussion by jnaav** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#199] [Q&A] [answered]
I am using this card with the  Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor (great components both of them).

When vacuum is docked, after a while, or goes to deep sleep, camera map goes unavailable. I am using the blueprint automation to disable the camera, but in the card, the map dissapears and then this message appears:
![imagen](https://user-images.githubusercontent.com/61027944/142772036-4c41e635-a50c-46e7-bb18-c1ddd0b3b100.png)

Is there any way not to see this "error" message in my dashboard? Maybe showing last retrieved image of the map or something like that. How do you handle this? 

When I turn on the vacuum it appears again and everything works great, but this is only 10% of the time. The rest of the time the vacuum is off and them the map dissapears. How do you deal with that? Am I missing something in my configuration maybe?

**Comment by PiotrMachowski** [comment, marked as answer]
I plan to handle this case in Map Extractor

**Reply by jnaav** [reply]
Great! Thanks for your effort!
